codes/c/chapter_divide_and_conquer/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_executable(hanota.c)
```

Comment:
The `add_executable` syntax is 
```cmake
add_executable(<name> [source1] [source2 ...])
```
Reference: https://cmake.org/cmake/help/latest/command/add_executable.html